### PR TITLE
refactor(rxEncoreRoutes): rx-prefixes update to encoreRoutes

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1166

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/rxEncoreRoutes/docs/rxEncoreRoutes.meta.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (halkon) [inline src/utilities/rxEncoreRoutes/docs/rxEncoreRoutes.meta.json]: forgot to `git mv` or is github being picky?

Comment (cathysiller) [inline src/utilities/rxEncoreRoutes/docs/rxEncoreRoutes.meta.json]: used `git mv` when moved directories from encoreRoutes to rxEncoreRoutes -- will fix :)

Comment (CITguy) [inline src/utilities/rxEncoreRoutes/docs/rxEncoreRoutes.meta.json]: I think this is just git being picky.

File: src/utilities/rxEncoreRoutes/scripts/rxEncoreRoutes.js [hunk 2]
```diff
@@ -74,5 +74,20 @@ angular.module('encore.ui.utilities')
...
+ * @deprecated
+ * Please use rxEncoreRoutes instead. This item will be removed on the 4.0.0 release.
+ * @ngdoc service
+ * @name utilities.service:encoreRoutes
+ * @requires utilities.service:rxEncoreRoutes
+ */
+.service('encoreRoutes', function (rxEncoreRoutes) {
+    console.warn (
+        'DEPRECATED: EncoreRoutes - Please use rxEncoreRoutes. ' +
+        'EncoreRoutes will be removed in EncoreUI 4.0.0'
```
Comment (CITguy) [inline src/utilities/rxEncoreRoutes/scripts/rxEncoreRoutes.js]: Please update all `EncoreRoutes` in the warning message to match the service name (lowerCamelCase, `encoreRoutes`).

Comment (cathysiller) [inline src/utilities/rxEncoreRoutes/scripts/rxEncoreRoutes.js]: updated "EncoreRoutes" to "encoreRoutes" in deprecation message

**Comment by CITguy** [comment]
@cathysiller: restart both Travis and Codeship tests on Thursday morning. Seems we haven't eliminated a UTC bug in rxDatePicker.

**Pull request merged**